ReText/editor.py

```diff
@@ -181,13 +181,28 @@ def contextMenuEvent(self, event):
 		actions = [self.parent.act(sug, trig=self.fixWord(sug)) for sug in suggestions]
 		menu = self.createStandardContextMenu()
 		menu.insertSeparator(menu.actions()[0])
+		menu.insertAction(menu.actions()[0], self.parent.act(self.tr('Add to dictionary'), trig=self.learnWord(word)))
```

Comment:
This line is too long, please break it into two smaller ones (PEP 8 says lines should be < 80 characters, but I am trying to at least keep them below 100).

Reply:
Done !